contracts/ColonyStorage.sol

```diff
@@ -44,6 +44,26 @@ contract ColonyStorage is DSAuth {
   // Pot 0 is the 'reward' pot containing funds that can be paid to holders of colony tokens in the future.
   mapping (uint256 => Pot) pots;
 
+  struct RewardPayoutCycle {
+    // Reputation root hash at the time of reward payout creation
+    bytes32 reputationState;
+    // Total tokens at the time of reward payout creation
+    uint256 totalTokens;
+    // Amount alocated for reward payout
+    uint256 amount;
+    // Remaining amount - decremented every time a user claims a payout
+    uint256 remaining;
+    // Token in which a reward is paid out with
+    address tokenAddress;
+    // Unix timestamp of when the payout has started
+    uint256 blockTimestamp;
+  }
+
+  mapping (uint256 => RewardPayoutCycle) rewardPayoutCycles;
+  mapping (address => bool) activeRewardPayouts;
+  uint8 globalRewardPayoutCount;
+  mapping (address => uint256) userRewardPayoutCount;
+
```

Comment:
Comment mappings, arrays, global variables, everything really that is storage. As you see all other items, not just structs in `ColonyStorage` are commented.
Our current stand on documentation is that `IColony` and `IColonyNetwork` are the two interfaces we document plus the two respective storage contracts: `ColonyStorage` and `ColonyNetworkStorage` 
In addition any other non-EtherRouter contracts (not covered by the above) are documented e.g. `ReputationMiningCycle`